### PR TITLE
Aliases mapping

### PR DESCRIPTION
**Pull request by stefan-lifebit**
**WHY?**
Currently we have to create alias manually. Goal is to automatically have new alias created on every commit.

**WHAT?**
- Updated script to create a new alias
- Refactored script using black & mypy

### CODE REVIEW

**Review by rishy — COMMENTED** [review]
We can make `alias_name` optional here. For actual staging and prod deployment, it would make sense to update that automatically, i.e. in `codeship-steps.yml` we can pass the value of `alias_name`, so the new version is tied to that alias.

The objective of `dev` deployment is to try multiple things. There can be a scenario, when someone is testing a currently deployed model, if it is using an alias `staging`, and someone else pushes a branch suffixed with `-dev`, the model in valohai will be updated automatically, which might not be desirable all the time.

Also, note that while `main` branch which deploys in `<MODEL_NAME>` deployment will usually have only one version running `staging/prod`. While for `dev` deployment, there can be multiple versions running for multiple branches suffixed with `-dev`. 


Let's discuss that during today's call before merging.

**Comment by stefan-lifebit** [comment]
> We can make `alias_name` optional here. For actual staging and prod deployment, it would make sense to update that automatically, i.e. in `codeship-steps.yml` we can pass the value of `alias_name`, so the new version is tied to that alias.
> 
> The objective of `dev` deployment is to try multiple things. There can be a scenario, when someone is testing a currently deployed model, if it is using an alias `staging`, and someone else pushes a branch suffixed with `-dev`, the model in valohai will be updated automatically, which might not be desirable all the time.
> 
> Also, note that while `main` branch which deploys in `<MODEL_NAME>` deployment will usually have only one version running `staging/prod`. While for `dev` deployment, there can be multiple versions running for multiple branches suffixed with `-dev`.
> 
> Let's discuss that during today's call before merging.

Actually, it is optional. You can call a script without alias_name argument. I created everything to work just like you said here, using codeship-steps.yml

File: valohai_deployment_action/valohai_deployment.py [hunk 1]
```diff
@@ -84,61 +83,74 @@ def create_version(
...
+    logging.info(response)
+
+    body = {
+        "deployment": DEPLOYMENT_ID,
+        "target": response["endpoints"][0]["version"],
+        "name": alias_name,
+    }
+
+    # Send a POST request to create a new alias for this deployment
+    deployment_response = requests.post(
```
[the remark this answers is not in the document]
Comment (stefan-lifebit) [reply valohai_deployment_action/valohai_deployment.py]: Fixed.

**Review by rishy — APPROVED** [review]
Lgtm!

**Pull request merged**